lms/resources/_js_config/__init__.py

```diff
@@ -483,6 +487,7 @@ def _blackboard_sync_api(self):
                 },
                 "assignment": {
                     "resource_link_id": self._context.lti_params["resource_link_id"],
+                    "group_set_id": self._context.group_set_id,
                 },
                 "group_info": {
```

Comment:
It would be lovely to have a comment for each of these sections saying why we pass them through. I can't see why we'd need the group info? Or is this just a weird way of passing a bunch of`LTI launch params?